AsyncDisplayKit/ASViewController.mm

```diff
@@ -82,6 +83,13 @@ - (void)loadView
   _node.frame = frame;
   _node.autoresizingMask = autoresizingMask;
   self.view = view;
+  
+  // ensure that self.node has a valid trait collection before a subclass's implementation of viewDidLoad.
+  // Any subnodes added in viewDidLoad will then inherit the proper environment.
+  if (AS_AT_LEAST_IOS8) {
+    ASEnvironmentTraitCollection traitCollection = [self displayTraitsForTraitCollection:self.traitCollection];
```

Comment:
The terminology here is not clear to me ("display traits") and should be considered with the comment above.  Is this essentially asyncTraitsForTraitCollection: or environmentTraitsForTraitCollection?